src/TestSuite/IntegrationTestTrait.php

```diff
@@ -491,8 +493,8 @@ public function options(array|string $url): void
      */
     protected function _sendRequest(array|string $url, string $method, array|string $data = []): void
     {
+        $url = $this->resolveUrl($url);
         $dispatcher = $this->_makeDispatcher();
-        $url = $dispatcher->resolveUrl($url);
```

Comment:
Are we duplicating the logic below and should somehow deprecated this or are these 2 different implementations on purpose?